driver/pace/driver/diagnostics.py

```diff
@@ -131,3 +156,26 @@ def store_grid(
         self, grid_data: pace.util.grid.GridData, metadata: QuantityMetadata
     ):
         pass
+
+
+def _compute_column_integral(
+    name: str, q_in: pace.util.Quantity, delp: pace.util.Quantity
+):
+    """
+    Compute column integrated mixing ratio (e.g., total liquid water path)
+
+    Args:
+        q_in: tracer mixing ratio
+        delp: pressure thickness of atmospheric layer
+    """
+    column_integral = pace.util.Quantity(
+        sum(
+            q_in.data[:, :, k] * delp.data[:, :, k]
+            for k in range(q_in.metadata.extent[2])
```

Comment:
do we need to check that the extents are the same?

Reply:
This is a sum over the k-axis, so it should error if you try to put a 2D variable.

Reply:
Will add error message